src/main/scala/org/broadinstitute/dsde/firecloud/utils/RestJsonClient.scala

```diff
@@ -49,33 +51,49 @@ trait RestJsonClient extends FireCloudRequestBuilding {
     }
 
     val finalPipeline = addCreds.map(creds => creds ~> pipeline).getOrElse(pipeline)
-    finalPipeline(req)
+    finalPipeline(req) map { res =>
+      if (label.nonEmpty && tick != -1) {
+        val elapsed = System.currentTimeMillis() - tick
+        perfLogger.info(perfmsg(label.get, elapsed))
+      }
+
+      res
+    }
   }
 
-  def authedRequestToObject[T](req: HttpRequest, compressed: Boolean = false, useFireCloudHeader: Boolean = false, connector: Option[ActorRef] = None)(implicit userInfo: WithAccessToken, unmarshaller: Unmarshaller[T], ers: ErrorReportSource): Future[T] = {
-    requestToObject(true, req, compressed, useFireCloudHeader, connector)
+  def authedRequestToObject[T](req: HttpRequest, compressed: Boolean = false, useFireCloudHeader: Boolean = false, connector: Option[ActorRef] = None, label: Option[String] = None)(implicit userInfo: WithAccessToken, unmarshaller: Unmarshaller[T], ers: ErrorReportSource): Future[T] = {
+    requestToObject(true, req, compressed, useFireCloudHeader, connector, label)
   }
 
-  def unAuthedRequestToObject[T](req: HttpRequest, compressed: Boolean = false, useFireCloudHeader: Boolean = false, connector: Option[ActorRef] = None)(implicit unmarshaller: Unmarshaller[T], ers: ErrorReportSource): Future[T] = {
+  def unAuthedRequestToObject[T](req: HttpRequest, compressed: Boolean = false, useFireCloudHeader: Boolean = false, connector: Option[ActorRef] = None, label: Option[String] = None)(implicit unmarshaller: Unmarshaller[T], ers: ErrorReportSource): Future[T] = {
     implicit val userInfo:WithAccessToken = null
-    requestToObject(false, req, compressed, useFireCloudHeader, connector)
+    requestToObject(false, req, compressed, useFireCloudHeader, connector, label)
   }
 
+  // zero usages in the codebase
```

Comment:
Remove the unused method?

Reply:
I wanted to keep this PR light and didn't feel like making functional changes - I can see wanting to use this method in the future.